Java/tests/src/test/java/com/microsoft/recognizers/text/tests/choice/BooleanModelTest.java

```diff
@@ -63,4 +66,22 @@ protected void recognizeAndAssert(TestCase currentCase) {
             throw new AssumptionViolatedException(nex.getMessage(), nex);
         }
     }
+
+    @Override
+    public void assertResults(TestCase currentCase, List<ModelResult> results, List<String> testResolutionKeys) {
+        List<ModelResult> expectedResults = readExpectedResults(ModelResult.class, currentCase.results);
+        Assert.assertEquals(getMessage(currentCase, "\"Result Count\""), expectedResults.size(), results.size());
+
+        IntStream.range(0, expectedResults.size())
+                .mapToObj(i -> Pair.with(expectedResults.get(i), results.get(i)))
+                .forEach(t -> {
+                    ModelResult expected = t.getValue0();
+                    ModelResult actual = t.getValue1();
+
+                    Assert.assertEquals(getMessage(currentCase, "typeName"), expected.typeName, actual.typeName);
+                    Assert.assertEquals(getMessage(currentCase, "text"), expected.text, actual.text);
+
+                    assertModel(expected, actual, currentCase, testResolutionKeys);
+                });
+    }
```

Comment:
This is not ok as this is also validated in the [AbstractTest](https://github.com/southworks/Recognizers-Text/blob/internal/southworks/java/improve-model-validations/improve-choice-model/Java/tests/src/test/java/com/microsoft/recognizers/text/tests/AbstractTest.java#L142).

`typeName` and `text` are already validated there, the problem is `start` & `end`. I think that the correct change is to remove those properties in the `AbstractTest`.

I mean, all the global props should be validated in the `AbstractTest` class (e.g. typename/text), but `start` and `end` are specific of each class.